bin/ci

```diff
@@ -74,8 +74,12 @@ lein-test() {
     lein test
 }
 
+if [ -z ${CIRCLE_BRANCH_REGEX+x} ]; then
+    CIRCLE_BRANCH_REGEX='^master|release-.+$'
```

Comment:
I'd personally prefer this to be broken out into separate clauses rather than a single regex personally, but other than this egregious bit of regex horror, 👍